FAQ/FAQ.tex

```diff
@@ -41,44 +41,61 @@ \section{Getting started}
 Where do I get the latest Squeak?
 \end{faq}
 \answer
-\url{ftp.squeak.org/current_development}
+\url{https://squeak.org/downloads/}
 \index{download}
 
 \begin{faq}
 Which \sq image should I use with this book?
 \end{faq}
 \answer
-We recommend that you use the image provided on the Squeak by Example web page.
-Visit \url{http://SqueakByExample.org} and look for ``Download Squeak'' in the sidebar.
+We recommend that you use the standard image provided on \url{https://squeak.org/downloads/}.
 You should also be able to use most other images, but you may find that the hands-on exercises behave differently in surprising ways.
-% ct TODO: Broken link!
 
-%\begin{faq}
-%Where is the Squeak ``Development Image''?
-%\end{faq}
-%\answer
-%\url{www.squeaksource.com/ImageForDevelopers}
-%This is a specially-prepared image including lots of useful packages pre-installed for developers.
+\begin{faq}
+I have destroyed my world docking bar at the top of the world in some way.
+How do I get it back?
+\end{faq}
+\answer
+Open the yellow button menu of the world (by clicking the yellow button in the world). 
+There you find the menu item \menu{show main docking bar}.
+Click it once to deactivate it completely, and then click it a second time to re-activate it.
+
+\begin{faq}
+Some code triggered a lot of debuggers and my whole screen is filled with them. 
+How can I get rid of them?
+\end{faq}
+\answer
+You can close all windows that look similar at once by selecting \menu{World docking bar \go Windows \go Bad Window \go Close all like this}.
```

Comment:
Just by interested, how is this different from Do > Utilities closeAllDebuggers? Is the latter redundant?

Reply:
I don't think there is any difference in the actual functionality, except that `close all like this` is more general, and I wanted to convey this more general way of managing windows. 
(Also, I guess `closeAllDebuggers` might be more robust against very large numbers of debuggers, as the list of menus requires quite a long time to build up in this case)

Reply:
And `closeAllDebuggers` is better reachable using the keyboard only (Ctrl + 5, d, Enter). However. :-)